resources/lang/fr/pireps.php

```diff
@@ -16,14 +16,14 @@
     'filedon'              => 'Déposé le',
     'fare'                 => 'Tarif|Tarifs',
     'class'                => 'Classe',
-    'count'                => 'Comte',
-    'flighttime'           => 'Temps de vol',
+    'count'                => 'Compte',
+    'flighttime'           => 'Durée du vol',
     'flightlevel'          => 'Niveau de vol',
     'fieldsreadonly'       => 'Une fois qu\'un PIREP est envoyé, certains champs passent en lecture seul.',
     'flightinformations'   => 'Informations du vol',
     'flightident'          => 'Numéro/Code/Etape du vol',
-    'block_fuel'           => 'Carburant emporté',
-    'fuel_used'            => 'Carburant utilisé',
+    'block_fuel'           => 'Fuel embarqué',
+    'fuel_used'            => 'Consommation',
```

Comment:
I think the previous ones were better since fuel is not french and consommation is more like "fuel flow" than "fuel used"

Reply:
"Carburant embarqué" is good,
"Carburant consommé" is a bit long for the display, I don't really like "Consommation"
If you have a suggest ...

Reply:
Carburant utilisé ?